### DISCUSSION TITLE
Episode 163: Sean Parent on Tilt Five, Metrowerks & Be Inc.

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#55] [Episode]
Episode Link: https://adspthepodcast.com/2024/01/05/Episode-163.html

Feel free to write comments or ask questions 🙂

**Comment by sasq64** [comment]
I can recommend the cppcast episode where they interview Jeri, its fantastic;
https://cppcast.com/jeri-ellsworth-tiltfive/

There's a great exchange between Jason and Jeri where they talk about the C64 DTV chips that she still have at home, something like:

"Yeah I actually have 250,000 of the chips in my closet right now."
"I'm sorry, did you say 250 to a thousand?"
"250 thousand."
